### PR TITLE
add option to display AvatarData receive stats with display name

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 62688e5c53d70399e2f908cebf2c224e9a8c7b94 available at: https://gist.github.com/986ee5d19bfeace27224<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4713.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4713.dmg

File: libraries/avatars/src/AvatarHashMap.cpp [hunk 1]
```diff
@@ -25,11 +25,11 @@ AvatarHash::iterator AvatarHashMap::erase(const AvatarHash::iterator& iterator)
     return _avatarHash.erase(iterator);
 }
 
-const qint64 AVATAR_SILENCE_THRESHOLD_MSECS = 5 * 1000;
+const qint64 AVATAR_SILENCE_THRESHOLD_USECS = 5 * 1000 * 1000;
```
Comment (AndrewMeadows) [inline libraries/avatars/src/AvatarHashMap.cpp]: Should probably be 5 * USECS_PER_SECOND.

File: libraries/shared/src/SimpleMovingAverage.cpp [hunk 2]
```diff
@@ -52,9 +52,9 @@ void SimpleMovingAverage::reset() {
 
 float SimpleMovingAverage::getEventDeltaAverage() const {
     return (ONE_MINUS_WEIGHTING * _eventDeltaAverage) +
-        (WEIGHTING * ((usecTimestampNow() - _lastEventTimestamp) / 1000000.0f));
+        (WEIGHTING * ((usecTimestampNow() - _lastEventTimestamp) / 1000000.0f ));
 }
```
Comment (AndrewMeadows) [inline libraries/shared/src/SimpleMovingAverage.cpp]: Extra whitespace near end of line should be removed.

File: libraries/shared/src/SimpleMovingAverage.h [hunk 3]
```diff
@@ -25,8 +25,11 @@ class SimpleMovingAverage {
     
     int getSampleCount() const { return _numSamples; };
     float getAverage() const { return _average; };
-    float getEventDeltaAverage() const;
-    float getAverageSampleValuePerSecond() const;
+    float getEventDeltaAverage() const; // returned in microseconds
```
Comment (AndrewMeadows) [inline libraries/shared/src/SimpleMovingAverage.h]: Comment says the getEventDeltaAverage() returns microseconds, but as far as I can tell the return value is actually in seconds.

**Comment by birarda** [comment]
@AndrewMeadows I moved this to #4722

**Pull request closed without merging**